cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc

```diff
@@ -0,0 +1,17 @@
+[id="cnv-monitoring-vm-health"]
+= Monitoring virtual machine health
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-monitoring-vm-health
+toc::[]
+
+Liveness probes stop the `VirtualMachineInstance` if it fails and allow
```

Comment:
I would prefer if the intro for this assembly were its own concept module, with a brief intro sentence in the assembly instead.

Reply:
Done.